ui/app/templates/error.hbs

```diff
@@ -0,0 +1,46 @@
+{{!
+  Copyright (c) HashiCorp, Inc.
+  SPDX-License-Identifier: BUSL-1.1
+~}}
```

Comment:
Is this header okay because it's just backported to 1.15?

Reply:
Yup, I checked in the codebase on this release and all the other .js files have this too